sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -245,7 +245,7 @@ statement
     | SET TIME ZONE timezone=(STRING | LOCAL)                          #setTimeZone
     | SET TIME ZONE .*?                                                #setTimeZone
     | SET .*?                                                          #setConfiguration
-    | RESET                                                            #resetConfiguration
+    | RESET multipartIdentifier?                                       #resetConfiguration
```

Comment:
How should we handle a hive session conf in `RESET`, e.g., `hivevar:xxx:yyy`? https://issues.apache.org/jira/browse/SPARK-18086

Reply:
Seems that we can directly use `RESET xxx` as `xxx` is stored in sql conf indeed

Reply:
should this be consistent with `SET .*?`?

Reply:
OK, sounds reasonable to me